extensions/2.0/Vendor/EXT_feature_metadata/0.0.0/README.md

```diff
@@ -0,0 +1,665 @@
+
+**Note**: in the core glTF standard, only PNG and JPEG textures are supported. This limits data types to those based on `UINT8`:
+
+* `UINT8`
+* normalized `UINT8` (the texture stores [0, 255], but values are remapped to [0, 1])
+* `ARRAY` of `UINT8` with `componentCount` of `1-4`
+* `ARRAY` of normalized `UINT8` with `componentCount` of `1-4`
+
+In this glTF extension, textures are defined with the following steps:
+1. A class is defined in the root `EXT_feature_metadata.classes` object. This is used to describe the metadata in the texture.
+2. A feature texture is defined in the root `EXT_feature_metadata.featureTextures` object. This must reference the class ID defined in step 1.
+3. A feature texture is associated with a primitive by listing the texture ID in the `primitive.EXT_feature_metadata.featureTextures` array.
+
+Example:
+
+```json
+{
+  "extensions": {
+    "EXT_Feature_metadata": {
+      "classes": {
+        "elevation": {
+          "name": "Elevation",
+          "properties": {
+            "elevation": {
+              "name": "Elevation (m)",
+              "description": "Elevation above the WGS84 ellipsoid"
+            }
+          }
+        }
+      },
+      "featureTextures": {
+        "elevationTexture": {
+          "class": "elevation",
+          "properties": {
+            "elevation": {
+              "texture": {
+                "index": 0,
+                "texCoord": 0
+              },
+              "channels": "r"
+            }
+          }
+        }
+      }
+    }
+  },
+  "meshes": [
+    {
+      "name": "Feature Texture Example",
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 0,
+            "NORMAL": 1,
+            "TEXCOORD_0": 2
+          },
+          "indices": 3,
+          "material": 0,
+          "extensions": {
+            "EXT_feature_metadata": {
+              "featureTextures": ["elevationTexture"]
+            }
+          }
+        }
+      ]
+    }
+  ],
+}
+```
+
+## Feature Identification
+
+The Cesium 3D Metadata Specification lets the implementation determine how to identify features. This glTF extension provides three different methods for associating vertices or texels with **feature IDs**.
+
+### Feature ID Accessors
+
+![Feature ID Accessor diagram](figures/feature-id-accessor.jpg)
+
+The most straightforward method for defining feature IDs is to store them directly in a buffer. In this extension, this is done by using a glTF attribute. Feature ID attributes must follow the naming convention `_FEATURE_ID_X` where `X` is a non-negative integer. The first feature ID attribute is `_FEATURE_ID_0`, the second `_FEATURE_ID_1`, and so on. 
+
+**Note**: Since glTF accessors do not support `UNSIGNED_INT` types, `FLOAT32` must be used instead. This allows for integer feature IDs up to `2e24`.
```

Comment:
`FLOAT32` should be `FLOAT` when we're talking about glTF accessors.

I don't think `FLOAT` is strictly required. You could use `UINT8` or `UINT16` but you'd need to add padding between each element so that each element is 4-byte aligned.